sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -713,6 +719,9 @@ PRECEDING: 'PRECEDING';
 FOLLOWING: 'FOLLOWING';
 CURRENT: 'CURRENT';
 ROW: 'ROW';
+EXCLUDE: 'EXCLUDE';
```

Comment:
Are these reserved keywords? If they are not please add them to the nonReserved rule.